_docs_integrate/send-forms-via-requests.md

```diff
@@ -0,0 +1,218 @@
+A Request can therefore contain both form fields and Attribute related RequestItems, such as for [reading Attributes from a peer]({% link _docs_integrate/read-attributes-from-peer.md %}).
+{: .notice--info}
+
+## Send and Receive the Request
+
+The Sender that wants to send a form to the Recipient may or may not already have a Relationship with the Recipient.
+Depending on which is the case, a different method can be used to send the [Request for a form]({% link _docs_integrate/send-forms-via-requests.md %}#request-for-a-form).
+There are two ways to send the Request for a form created by the Sender to the Recipient.
+
+### Request via RelationshipTemplate
+
+If there is currently no Relationship between the Sender and the Recipient, this approach must be used.
+But it is also possible for the Sender to use a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) to send a Request to the Recipient if there is already an active Relationship between them.
+All details on how to send and receive a Request via a RelationshipTemplate in general can be found in the [Requests via RelationshipTemplates]({% link _docs_integrate/requests-via-relationshiptemplates.md %}) guide.
+
+### Request via Message
+
+The Sender only has the option of sending a Request to the Recipient via a [Message]({% link _docs_integrate/data-model-overview.md %}#message) if there is already an active Relationship between them.
+All information on how to send and receive a Request via a Message can be found in the [Requests via Messages]({% link _docs_integrate/requests-via-messages.md %}) guide.
+
+## Accept the Request
+
+After the Recipient has received the [Request for a form]({% link _docs_integrate/send-forms-via-requests.md %}#request-for-a-form), it can accept it to fill out the form sent by the Sender.
+To do this, proceed as described in the [Accept incoming Request]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}) use case documentation and specify the `id` of the received [Request]({% link _docs_integrate/data-model-overview.md %}#request).
+It must also be decided and specified for each FormFieldRequestItem contained in the Request for a form whether it should be accepted or rejected.
+
+If the Recipient does not want to fill out any form field of the form sent by the Sender and, therefore, does not want to accept the Request for a form of the Sender, it can reject it as a whole as well.
+For this, follow the instructions of the [Reject incoming Request]({% link _docs_use-cases/use-case-consumption-reject-incoming-request.md %}) use case.
+{: .notice--info}
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/35209f55-89ed-4bff-96e5-61021fb74edf" id="AZa_znY.QyPU"></iframe></div>
```

Comment:
Hmm, I don't like that fact that you write about http requests in this diagram. IMO this layer can be removed.